sycl/test-e2e/bindless_images/read_1D.cpp

```diff
@@ -1,7 +1,7 @@
-// REQUIRES: cuda
+// REQUIRES: cuda || (level_zero && gpu-intel-dg2)
 
 // RUN: %{build} -o %t.out
-// RUN: %t.out
+// RUN: env NEOReadDebugKeys=1 UseBindlessMode=1 UseExternalAllocatorForSshAndDsh=1 %t.out
```

Comment:
How come only this test requires these environment variables? Maybe we should add them to the LIT environment for all Bindless tests?

Reply:
these env are required for all tests in this PR